src/governance.cpp

```diff
@@ -172,7 +175,10 @@ void CGovernanceManager::ProcessMessage(CNode* pfrom, std::string& strCommand, C
     else if (strCommand == NetMsgType::MNGOVERNANCEOBJECTVOTE)
     {
         // Ignore such messages until masternode list is synced
-        if(!masternodeSync.IsMasternodeListSynced()) return;
+        if(!masternodeSync.IsMasternodeListSynced()) {
+            LogPrintf("CGovernanceManager::ProcessMessage MNGOVERNANCEOBJECTVOTE -- masternode list not synced\n");
```

Comment:
I don't think it should because we don't start syncing governance until the masternode list is synced.  So I think this is more of an abnormal condition.